javalanglib/src/main/scala/java/lang/System.scala

```diff
@@ -6,7 +6,35 @@ import scala.scalajs.js
 import js.Dynamic.global
 import scala.scalajs.LinkingInfo.assumingES6
 
+import java.{util => ju}
+import scalajs.runtime.environmentInfo
+
 object System {
+
+  private[this] var systemProperties = loadSystemProperties()
+
+  private def loadSystemProperties(): ju.Properties = {
+    val sysProp = new ju.Properties()
+    sysProp.setProperty("java.version", "1.8")
+    sysProp.setProperty("java.vm.specification.version", "1.8")
+    sysProp.setProperty("java.vm.specification.vendor", "Oracle Corporation")
+    sysProp.setProperty("java.vm.specification.name", "Java Virtual Machine Specification")
```

Comment:
I do set it, but the code is in another place because I load it directly from the plugin:
https://github.com/scala-js/scala-js/pull/2237/files#diff-f74255631d0aae4a116f32f2cb6385ccR643